src/Models/ShallowWaterModels/shallow_water_model.jl

```diff
@@ -70,7 +72,9 @@ function ShallowWaterModel(;
                              tracers = (),
                        diffusivities = nothing,
      boundary_conditions::NamedTuple = NamedTuple(),
-                 timestepper::Symbol = :RungeKutta3)
+                 timestepper::Symbol = :RungeKutta3
+                   immersed_boundary = NoImmersedBoundary(),
```

Comment:
```suggestion
                 timestepper::Symbol = :RungeKutta3
```

Reply:
I'm happy to make this change.  For some reason, I can't commit suggestion. I guess that means I need to change it and push the change?

Reply:
Just pushed the change